docs/v3/signature_changes.md

```diff
@@ -9,21 +9,21 @@ sidebar_position: 1
 
 XMTP v2 signatures use `secp256k1` keys signed by the user’s wallet. To use MLS for group chat, XMTP is migrating to `ed25519` keys signed by the `secp256k1` keys.
 
-XMTP is also requiring a signature for each app/device combination (“installation”).  Users will have to authorize XMTP for each app they install on each device (instead of once per app).
+XMTP is also requiring a signature for each app/device combination (“installation”).  This is a change from requiring a signature just for users.  Users will have to authorize XMTP for each app they install on each device (instead of once per app).
 
 ## How do XMTP Identities get the new key type?
 
-When an existing user opens a V3 app for the first time, XMTP will take their `secp256k1` Create Identity signature and sign it with an `ed25519` key.
+When an existing user opens a V3 app for the first time, XMTP will take their `ed25519` Create Identity signature and sign it with an `secp256k1` key.
 
-New users who go through the Create Identity signature process will also get an `secp256k1` Create Identity signature signed with an `ed25519` key.
+New users who go through the Create Identity signature process will also get an `ed25519` Create Identity signature signed with an `secp256k1` key.
```

Comment:
The `ed25519` isn't signed via `Create Identity`, the `secp256k1` is. I would modify to:

> an `ed25519` key signed with their `secp256k1` key from the Create Identity process.

or similar, here and above